Source/WebCore/platform/graphics/ca/win/PlatformCAAnimationWin.cpp

```diff
@@ -539,13 +539,12 @@ void PlatformCAAnimationWin::copyKeyTimesFrom(const PlatformCAAnimation& value)
 
 void PlatformCAAnimationWin::setTimingFunctions(const Vector<Ref<const TimingFunction>>& value, bool reverse)
```

Comment:
I wish we had named `value` to be `timingFunctions`.

Reply:
I almost made that change, but `PlatformCAAnimationCocoa.mm` also used `value`, so I left it alone for consistency.

Reply:
Posted PR for this here:  https://github.com/WebKit/WebKit/pull/4315

Landed here:
Committed [254460@main](https://commits.webkit.org/254460@main) (c020d23d66d1): <https://commits.webkit.org/254460@main>